services/121-service/src/cronjob/cronjob.controller.ts

```diff
@@ -176,8 +176,9 @@ export class CronjobController {
     console.log('cronJobMethodNames: ', cronJobMethodNames);
     const responses: RunCronjobsResponseDto[] = [];
     for (const cronJobMethodName of cronJobMethodNames) {
-      const response =
-        await this.cronjobInitiateService[cronJobMethodName](cronJobMethodName);
+      const response = await (this.cronjobInitiateService as any)[
```

Comment:
I would avoid using `any` explicitly as much as possible. Is there a way to avoid that here?